### PR TITLE
Crashlytics add to subspec and Firebase.h

### PR DESCRIPTION
**Pull request by samedson**
This will allow developers to include `pod 'Firebase/Crashlytics'` and import it with `@import Firebase;`

### CODE REVIEW

File: Firebase.podspec [hunk 1]
```diff
@@ -69,6 +69,11 @@ Simplify your iOS development, grow your user base, and monetize more effectivel
     ss.dependency 'FirebaseAuth', '~> 6.4.1'
   end
 
+  s.subspec 'Crashlytics' do |ss|
+    ss.dependency 'Firebase/CoreOnly'
+    ss.ios.dependency 'FirebaseCrashlytics', '~> 4.0.0-beta.1'
```
Comment (paulb777) [inline Firebase.podspec]: Any reason that tvos and macos are excluded?

Comment (samedson) [reply Firebase.podspec]: Let me add them

File: Firebase.podspec [hunk 2]
```diff
@@ -72,6 +72,8 @@ Simplify your iOS development, grow your user base, and monetize more effectivel
   s.subspec 'Crashlytics' do |ss|
     ss.dependency 'Firebase/CoreOnly'
     ss.ios.dependency 'FirebaseCrashlytics', '~> 4.0.0-beta.1'
+    ss.osx.dependency 'FirebaseCrashlytics', '~> 4.0.0-beta.1'
+    ss.tvos.dependency 'FirebaseCrashlytics', '~> 4.0.0-beta.1'
```
Comment (paulb777) [inline Firebase.podspec]: Just one dependency without a platform modifier.

 `ss.dependency 'FirebaseCrashlytics', '~> 4.0.0-beta.1'`

File: Firebase.podspec [hunk 3]
```diff
@@ -69,6 +69,13 @@ Simplify your iOS development, grow your user base, and monetize more effectivel
     ss.dependency 'FirebaseAuth', '~> 6.4.1'
   end
 
+  s.subspec 'Crashlytics' do |ss|
+    ss.dependency 'Firebase/CoreOnly'
+    ss.ios.dependency 'FirebaseCrashlytics', '~> 4.0.0-beta.1'
+    ss.osx.dependency 'FirebaseCrashlytics', '~> 4.0.0-beta.1'
+    ss.tvos.dependency 'FirebaseCrashlytics', '~> 4.0.0-beta.1'
+  en
```
Comment (paulb777) [inline Firebase.podspec]: missing d

**Pull request merged**